packages/block-library/src/template-part/edit/placeholder/index.js

```diff
@@ -15,26 +15,34 @@ import { store as coreStore } from '@wordpress/core-data';
 import TemplatePartSelection from '../selection';
 
 export default function TemplatePartPlaceholder( {
+	area,
 	setAttributes,
 	innerBlocks,
 } ) {
 	const { saveEntityRecord } = useDispatch( coreStore );
 	const onCreate = useCallback( async () => {
 		const title = __( 'Untitled Template Part' );
+		const record = {
+			title,
+			slug: 'template-part',
+			content: serialize( innerBlocks ),
+		};
+		// If we have `area` set from block attributes, means an exposed
```

Comment:
The comments are very helpful, thanks for adding them

Reply:
This is how the area is sourced:
https://github.com/WordPress/gutenberg/blob/a5395f8e4a88b59816af1753a558c0e1208430bb/packages/block-library/src/template-part/index.php#L54-L57

It feels like we should make it work similar to the meta that is stored in `attributes` and saved together with the block. @mcsf, what do you think?